cmd/service-controller/controller.go

```diff
@@ -176,6 +176,22 @@ func NewController(cli *client.VanClient, origin string, tlsConfig *tls.Config,
 	}
 	AddStaticPolicyWatcher(controller.policy)
 
+	siteCreationTime := uint64(time.Now().UnixNano()) / uint64(time.Microsecond)
+	configmap, err := kube.GetConfigMap(types.SiteConfigMapName, cli.Namespace, cli.KubeClient)
+	if err == nil {
+		siteCreationTime = uint64(configmap.ObjectMeta.CreationTimestamp.UnixNano()) / uint64(time.Microsecond)
+	}
+	siteConfig, _ := controller.vanClient.SiteConfigInspect(context.TODO(), configmap)
+
+	var ttl time.Duration
+	var disableEvents = true
```

Comment:
In the RouterCreate the default is set to false.
Shouldn't it be set to false by default here as well?

Reply:
You were right, but it is not going to be necessary because I've removed that flag.